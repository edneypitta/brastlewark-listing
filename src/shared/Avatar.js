import React from 'react'
import PropTypes from 'prop-types'
import './Avatar.css'

const Avatar = props =>
  <img {...props} />

Avatar.propTypes = {
  src: PropTypes.string.isRequired,
  alt: PropTypes.string.isRequired
}

export default Avatar