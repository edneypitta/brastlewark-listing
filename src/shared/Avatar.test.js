import React from 'react'
import { shallow } from 'enzyme'
import Avatar from './Avatar'

describe('Avatar', () => {
  it('renders without crashing', () => {
    const component = shallow(<Avatar src="" alt="" />)
    expect(component).toMatchSnapshot()
  })
})
