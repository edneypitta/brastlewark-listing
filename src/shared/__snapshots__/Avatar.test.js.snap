// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Avatar renders without crashing 1`] = `
<img
  alt=""
  src=""
/>
`;
